rubin_sim/maf/run_comparison/gather_summaries.py

```diff
@@ -1,16 +1,99 @@
 #!/usr/bin/env python
 
 import glob
+import os
 import argparse
+import numpy as np
+import pandas as pd
+import sqlite3
+from rubin_sim.maf.db import ResultsDb
 
-from .run_comparison import RunComparison
+
+__all__ = ['gs', 'gather_summaries']
+
+
+def dirname_to_runname(inpath, replaces=["_glance", "_sci", "_meta", "_ss", "_ddf"]):
+    """Given a directory path, construct a plausible runname"""
+    result = os.path.basename(os.path.normpath(inpath))
+    for rstring in replaces:
+        result = result.replace(rstring, "")
+    return result
+
+
+def gs(run_dirs, dbfilename="resultsDb_sqlite.db"):
+    """Helper function for gather_summaries
+
+    Parameters
+    ----------
+    run_dirs : list of str
+        A list of directories to search for MAF result databases.
+    dbfilename : str (resultsDb_sqlite.db)
+        The database filename to look for (default: resultsDb_sqlite.db)."""
+
+    db_files = []
+    run_names = []
+    for dname in run_dirs:
+        fname = os.path.join(dname, dbfilename)
+        if os.path.isfile(fname):
+            db_files.append(fname)
+            run_names.append(dirname_to_runname(dname))
+
+    # querry to grab all the summary stats
+    sql_q = "SELECT metrics.metric_name, metrics.metric_info_label, "
+    sql_q += "metrics.slicer_name, summarystats.summary_name, "
+    sql_q += "summarystats.summary_value "
+    sql_q += "FROM summarystats INNER JOIN metrics ON metrics.metric_id=summarystats.metric_id"
+
+    rows = []
+    for row_name, fname in zip(run_names, db_files):
+        con = sqlite3.connect(fname)
+        temp_df = pd.read_sql(sql_q, con)
+        con.close()
+
+        # Use maf.db.ResultDb.build_summary_name so it matches show_maf labels
```

Comment:
Oh ha. I really thought that the contents on the show maf pages matched this, but when I double checked, they don't. 
So forget about it and just do whatever thing you were doing before.
Eventually we'll update all of the downstream stuff and dashboards anyway.